exiftool.go

```diff
@@ -144,14 +147,17 @@ func (e *Exiftool) ExtractMetadata(files ...string) []FileMetadata {
 	for i, f := range files {
 		fms[i].File = f
 
-		if _, err := os.Stat(f); err != nil {
+		s, err := os.Stat(f)
+		if err != nil {
+			fms[i].Err = err
 			if os.IsNotExist(err) {
 				fms[i].Err = ErrNotExist
-				continue
 			}
+			continue
+		}
 
-			fms[i].Err = err
-
+		if s.IsDir() {
```

Comment:
Seems like a sensible behavior for the api.

Reply:
I don't understand what you mean, sorry :p

Reply:
I was commenting here in response to your request for feedback on this solution. I think your solution is a good way to respond when a directory path is passed in as an argument.